finalPrep/annotations.js

```diff
@@ -2,41 +2,66 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+//declare a function called buildABear that takes in five parameters
 function buildABear(name, age, fur, clothes, specialPower) {
+  // declare a variable, greeting, and assigns a value which interpolates a string greeting with parameter name
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+  //declare a variable, demographics, which is assigned a array value that holds parameters name and age
   var demographics = [name, age];
+  //declare variable powerSaying and assigns in a string value that concatinates a string with parameter specialPower
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+  //declare a variable builtBear and assigns in it a value of an object
   var builtBear = {
+    //assign key basicInfo to value of parameter demographics
     basicInfo: demographics,
+    //assign key clothes to value of paramter clothes
     clothes: clothes,
+    //assign key exterior to value of parameter fur
     exterior: fur,
+    //assign key cost the value of 49.99
     cost: 49.99,
+    //assign key sayings the value of an array which holds the parameters greeting, powerSaying, and a string "Goodnight my friend!"
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
+    //assign key isCuddly a boolean value of true
     isCuddly: true,
   };
-
+  //returns the object builtBear
   return builtBear
 }
 
+//calls the funtion buildABear with the arguments 'Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares'
 buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
+//calls the function buildABear with the arguments 'Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in'
```

Comment:
Very nice job describing your keys and values, as well as recognizing where the function is called.